common.blocks/demo/demo.js

```diff
@@ -35,9 +35,14 @@ modules.define('demo', [ 'i-bem__dom', 'pretty', 'functions__debounce' ], functi
         _getBEMJSON: function() {
             return this._bemjson.getValue();
         },
+        changeEngine: function(data){
```

Comment:
```js
changeEngine: function(name) {
```

Please, rename argument to `name` and add space before `{` to be in project’s codestyle.